src/System.Runtime.InteropServices/src/redist/System.Runtime.InteropServices.depproj

```diff
@@ -5,8 +5,12 @@
   </PropertyGroup>
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
-    <AssemblyVersion>4.0.20.0</AssemblyVersion>
+    <AssemblyVersion>4.1.0.0</AssemblyVersion>
+    <AssemblyVersion Condition="'$(TargetGroup)' == 'netcore50' OR '$(TargetGroup)' == 'netcore50aot'">4.0.20.0</AssemblyVersion>
```

Comment:
Does the depproj even need the AssemblyVersion metadata? Is it only for the packaging validation?